src/generators/legacy-html/assets/style.css

```
@@ -0,0 +1,1137 @@
+/* --------------------- CSS Variables ---------------------------- */
```

```suggestion
/* --------------------- CSS Variables ---------------------------- */
```

Good practice is not to explain elements of a programming language. In this case, you shouldn't say it's a CSS variable.

---

You are aware I only copied the existing assets from node/node right?

---

Like, what's your point here? 😅

---

Yeah, so it's the moment to update it

---

No, Augustin, we won't be touching or spending time on the assets of the legacy HTML pages; The purpose here is (of this PR) is to bring the old legacy api docs into the new api docs tooling into a functional state.

Not make nitpick adjustments on these old assets that are not going to be updated as we will start working on the new stuff.

---

Also this is the most, genuinely speaking, irrelevant nitpick that you could have provided here 😅